src/packaging/metadata.py

```diff
@@ -635,6 +643,33 @@ def _process_requires_dist(
         else:
             return reqs
 
+    def _process_license_expression(self, value: str) -> str | None:
+        try:
+            return licenses.normalize_license_expression(value)
+        except ValueError as exc:
+            raise self._invalid_metadata(
+                f"{value!r} is invalid for {{field}}", cause=exc
+            ) from exc
+
+    def _process_license_files(self, value: list[str]) -> list[str]:
+        paths = []
+        for path in value:
+            if ".." in path:
+                raise self._invalid_metadata(
+                    f"{path!r} is invalid for {{field}}, "
+                    "parent directory indicators are not allowed"
+                )
+            if "*" in path:
+                raise self._invalid_metadata(
+                    f"{path!r} is invalid for {{field}}, " "paths must be resolved"
+                )
+            if path.startswith("/"):
```

Comment:
You could use `pathlib.PurePath.is_absolute()`.
```suggestion
            if pathlib.PurePosixPath(path).is_absolute() or pathlib.PureWindowsPath(path).is_absolute():
```

Reply:
46a7491894faf50d55ff7812f9ad3b61779b8422